pkg/sourcer/doc.go

```diff
@@ -1,2 +1,104 @@
 // Package sourcer implements the server code for User Defined Sourcer in golang.
+//
+// Example:
```

Comment:
you can just make them point to the source code in the examples dir, the code is bound to diverge.

Reply:
Agreed. Will do.